MuteSpotifyAds/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -1,63 +1,63 @@
 {
-  "images": [
+  "images" : [
     {
-      "size": "16x16",
-      "idiom": "mac",
-      "filename": "icon_16.png",
-      "scale": "1x"
+      "size" : "16x16",
+      "idiom" : "mac",
+      "filename" : "icon_16.png",
+      "scale" : "1x"
     },
     {
-      "idiom": "mac",
-      "size": "16x16",
-      "scale": "2x"
+      "idiom" : "mac",
+      "size" : "16x16",
+      "scale" : "2x"
     },
     {
-      "idiom": "mac",
-      "size": "32x32",
-      "filename": "icon_32.png",
-      "scale": "1x"
+      "size" : "32x32",
+      "idiom" : "mac",
+      "filename" : "icon_32.png",
+      "scale" : "1x"
     },
     {
-      "idiom": "mac",
-      "size": "32x32",
-      "scale": "2x"
+      "idiom" : "mac",
+      "size" : "32x32",
+      "scale" : "2x"
     },
     {
-      "idiom": "mac",
-      "size": "128x128",
-      "filename": "icon_128.png",
-      "scale": "1x"
+      "size" : "128x128",
+      "idiom" : "mac",
+      "filename" : "icon_128.png",
+      "scale" : "1x"
     },
     {
-      "idiom": "mac",
-      "size": "128x128",
-      "scale": "2x"
+      "idiom" : "mac",
+      "size" : "128x128",
+      "scale" : "2x"
     },
     {
-      "idiom": "mac",
-      "size": "256x256",
-      "filename": "icon_256.png",
-      "scale": "1x"
+      "size" : "256x256",
+      "idiom" : "mac",
+      "filename" : "icon_256.png",
+      "scale" : "1x"
     },
     {
-      "idiom": "mac",
-      "size": "256x256",
-      "scale": "2x"
+      "idiom" : "mac",
+      "size" : "256x256",
+      "scale" : "2x"
     },
     {
-      "idiom": "mac",
-      "size": "512x512",
-      "filename": "icon_512.png",
-      "scale": "1x"
+      "size" : "512x512",
+      "idiom" : "mac",
+      "filename" : "icon_512.png",
+      "scale" : "1x"
     },
     {
-      "idiom": "mac",
-      "size": "512x512",
-      "scale": "2x"
+      "idiom" : "mac",
+      "size" : "512x512",
+      "scale" : "2x"
     }
   ],
-  "info": {
-    "version": 1,
-    "author": "xcode"
+  "info" : {
```

Comment:
Furthermore, please do not update the binary images.

Reply:
Also did this because using git clone somehow managed to pull 0-bytes long icon files, and i did manually fix them by downloading them with wget so it could build, so these can be left out too